File: src/3295.report-spam-message.3541/solution.java
class Solution {
    public boolean reportSpam(String[] message, String[] bannedWords) {
        Set<String> set = new HashSet<>(List.of(bannedWords));
        int count = 0;
        for (String s : message) {
            if (set.contains(s)) {
                count++;
                if (count == 2) {
                    return true;
                }
            }
        }
        return false;
    }
}

File: src/3295.report-spam-message.3541/solution.ts
function reportSpam (message: string[], bannedWords: string[]): boolean {
    const set = new Set(bannedWords);
    let count = 0;
    for (let i = 0; i < message.length; i++) {
        if (set.has(message[i])) {
            count++;
            if (count === 2) {
                return true;
            }
        }
    }
    return false;
}
